### PR TITLE
Integrating Margins with Pano with tags code commented out

### PR DESCRIPTION
**Pull request by PrateekKuk**

### CODE REVIEW

File: js/values.js [hunk 1]
```diff
@@ -48,21 +51,38 @@ function highlighting(user, baseUrl) {
...
+        //         + "<script>$(function() { $.scrollDepth();});</script>");
+
       if (highlighting_enabled) {
         reenable_highlighting();
       }
-
       // showAllComments();
     }();
 
+  //kept giving me scroll depth function not found
```
Comment (amyxzhang) [inline js/values.js]: do we need to keep this commented out code? (also scrolldepth might be a non-core jquery library that needs to be added before you can call it)

File: js/values.js [hunk 2]
```diff
@@ -159,52 +181,68 @@ function highlighting(user, baseUrl) {
...
       var add_custom_tag_tags = $("<div>", {"class": "highlight-add-custom-tag-tags"});
       var add_valuetag_tag;
-      var highlight_itself_wrapper = $("<div>", {"class": "highlight-itself-wrapper"})
+      var highlight_itself_wrapper = $("<div>", {"class": "highlight-itself-wrapper", "style":"margin-top:5px"})
       var highlight_itself = $("<div>", {"class": "highlight-itself temp"});
+      
       highlight_itself.html(text);
       highlight_itself_wrapper.append(highlight_itself)
 
+      /* recent: the extra list of tags, don't need this, only need custom
```
Comment (amyxzhang) [inline js/values.js]: again, do we need this commented out code?

File: js/values.js [hunk 3]
```diff
@@ -159,52 +181,68 @@ function highlighting(user, baseUrl) {
...
-          });
+            });
+            add_valuetag_tag.html(t);
+            add_custom_tag_tags.append(add_valuetag_tag);
+          }
 
-          add_valuetag_tag.html(t);
-          add_custom_tag_tags.append(add_valuetag_tag);
-        }
+        //} recent: don't need this if i'm taking out generated
```
Comment (amyxzhang) [inline js/values.js]: I don't understand this comment?

File: js/values.js [hunk 4]
```diff
@@ -159,52 +181,68 @@ function highlighting(user, baseUrl) {
...
 
-        add_valuetag_tag.html(t);
-        add_valuetag_tags.append(add_valuetag_tag);
       }
-
+      */
       for (var t in all_tags) {
-        if (!(t in generated_tags)) {
-          add_valuetag_tag = $("<div>", {
+        //if (!(t in generated_tags)) {
```
Comment (amyxzhang) [inline js/values.js]: ditto to above

File: js/values.js [hunk 5]
```diff
@@ -159,52 +181,68 @@ function highlighting(user, baseUrl) {
...
-          add_valuetag_tag.html(t);
-          add_custom_tag_tags.append(add_valuetag_tag);
-        }
+        //} recent: don't need this if i'm taking out generated
       }
 
-      highlight_add_valuetag_header.html("What frames are used in this highlight?");
-      var highlight_add_valuetag_suggested = $("<div>", {"class": "highlight-add-valuetag-suggested light"});
-      highlight_add_valuetag_suggested.html("Suggested tags");
+      //var custom_tag_plus = $("<div>",{"class":"custom-tag-plus deselected highlight-tag","title":"Add a custom tag."});
```
Comment (amyxzhang) [inline js/values.js]: ditto

File: js/values.js [hunk 6]
```diff
@@ -267,6 +312,20 @@ function highlighting(user, baseUrl) {
     $(window).scroll(function(e) {
       var top = annote_position.top - ($(window).scrollTop() - annote_position.anchor_top);
       var left = annote_position.left - ($(window).scrollLeft() - annote_position.anchor_left);
+      
+      //this gets how far down the page you've gone -
```
Comment (amyxzhang) [inline js/values.js]: ditto

File: js/values.js [hunk 7]
```diff
@@ -267,6 +312,20 @@ function highlighting(user, baseUrl) {
...
+      // console.log($(window).height());
+      // console.log($(document).height());
+      
+      var scrolledDistance = $(window).scrollTop();
+      var windowHeight = $(window).height();
+      var totalHeight = $(document).height();
+
+      var totalScrolledPercent = (scrolledDistance / (totalHeight - windowHeight) ) *100;
+
+      console.log("% scrolled:" + totalScrolledPercent)
```
Comment (amyxzhang) [inline js/values.js]: let's not have console logs in production unless you really need it

File: js/values.js [hunk 8]
```diff
@@ -354,26 +413,28 @@ function highlighting(user, baseUrl) {
             should_highlight = false;
           }
 
-          // Ensure only trying to highlight in a text block
+          // Ensure only trying to highlight in a text block - may need to comment this out
           if (!$(e.target).is("p") && !$(e.target).is("div")) {
             if ($(e.target).is("em, strong, li, ul, ol, b, span")) {
               if (!$(e.target).parent().is("p, div")) {
-                should_highlight = false;
+                should_highlight = false; //
```
Comment (amyxzhang) [inline js/values.js]: What are these // that are at the end of lines?

File: js/values.js (hunk 1 above)
Comment (amyxzhang) [reply js/values.js]: I see it's included below. Hm I wonder if maybe it hasn't been imported yet in the code?

File: js/values.js (hunk 8 above)
Comment (PrateekKuk) [reply js/values.js]: Whoops sorry, I thought I took them all out. These were just my way of flagging where I had changed something. I'll go through this again and remove the extra comments and the "//"

File: js/values.js (hunk 1 above)
Comment (PrateekKuk) [reply js/values.js]: ohh I see what happened, my latest push got added to the pull request. The scroll depth code is not ready for production. It's not connected to the server yet

File: js/values.js (hunk 7 above)
Comment (PrateekKuk) [reply js/values.js]: agreed, will take these out during cleanup of comments. Didn't realize my latest push would update the pull request

**Pull request merged**